networking/network_observability/network-observability-overview.adoc

```diff
@@ -36,6 +36,7 @@ The Network Observability Operator provides the Flow Collector API custom resour
 
 The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
 
+Additionally, in *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overview of the persistent contents of your network. You can view metrics, such as *Top 5 flow rates per minute per namespace*, *Top 5 Ingress bandwidth per source and destination nodes*, and *Top 5 Ingress bandwidth per source and destination namespace*. Also in *Observe* -> *Dashboards*, the *Netobserv/Health* dashboard provides metrics about the health of the Operator.
```

Comment:
> a quick overview of the persistent contents of your network

not sure what are we trying to say here? would it be best say simply `provides a quick overview of the network flows in your {product} cluster`

Reply:
I think I got this specific piece directly from a Jira. However, I can see how what you suggest is more specific. I guess I was thinking that the "persistent contents" of the network were the listed metrics. But I guess also these are all network flow-based, so what you suggest makes sense too.